content/blog/haven-2-4-0/index.Rmd

```diff
@@ -0,0 +1,70 @@
+---
+output: hugodown::hugo_document
+
+slug: haven-2-4-0
+title: haven 2.4.0
+date: 2021-04-15
+author: Hadley Wickham
+description: >
+    This version provides much improved `labelled_spss()` support, improved
+    date-time handling, latest ReadStat, and a bunch of other small 
```

Comment:
```suggestion
    date-time handling, the latest ReadStat, and a bunch of other small 
```

Reply:
I think this is correct given the other two list items are gerunds, and you have an article (a) before the last item which is effectively a mass noun.